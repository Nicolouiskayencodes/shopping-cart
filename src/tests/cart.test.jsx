import { describe, it, expect } from "vitest";
import { render, screen } from "@testing-library/react";
import Cart from "../components/cart";

describe("Cart component", () => {
  it("renders correct heading", () => {
    render(<Cart items={[]}/>);
    expect(screen.getAllByRole("heading")[0].textContent).toMatch(/Shopping Cart/i);
  });
  it("renders cart", ()=>{
    const { container } = render(<Cart items={[]}/>);
    expect(container).toMatchSnapshot();
  })
});