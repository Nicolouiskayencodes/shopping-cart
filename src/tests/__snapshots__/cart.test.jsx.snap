// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Cart component > renders cart 1`] = `
<div>
  <div
    class="_body_d97d86"
  >
    <h1
      class="_title_d97d86"
    >
      Shopping Cart
    </h1>
    <p
      class="_empty_d97d86"
    >
      Your cart is empty!
    </p>
  </div>
</div>
`;
